agent/lib_aws_sdk_php.h

```diff
@@ -7,6 +7,21 @@
 #ifndef LIB_AWS_SDK_PHP_HDR
 #define LIB_AWS_SDK_PHP_HDR
 
+#if ZEND_MODULE_API_NO >= ZEND_8_1_X_API_NO /* PHP8.1+ */
+/* Service instrumentation only supported above PHP 8.1+*/
+
+/* SQS */
+#define SQS_LIBRARY_NAME "SQS"
+#define AWS_SQS_MESSAGING_SERVICE "aws_sqs"
```

Comment:
This is not used anywhere.

Reply:
Could be used [here](https://github.com/newrelic/newrelic-php-agent/blob/feat/sqs_instrumentation/agent/lib_aws_sdk_php.c#L95) and [here](https://github.com/newrelic/newrelic-php-agent/blob/feat/sqs_instrumentation/agent/lib_aws_sdk_php.c#L97).

Reply:
Thanks for catching, that was the intention.
Please see: [8a45142](https://github.com/newrelic/newrelic-php-agent/pull/1003/commits/8a451426aa9dc8ada6afa1ece55afee6b7d6f564)